REST/Documentation.json

```diff
@@ -6011,14 +6103,14 @@
       },
       "parameters": []
     },
-    "/applications/{applicationid:int}/users/{userid:int}/fuelconsumption": {
+    "/applications/{applicationid:int}/accumulators/{accumulatorid:long}": {
       "get": {
         "tags": [
-          "FuelConsumption"
```

Comment:
Fuel consumption seems to be gone?

Reply:
Hmm, I'm fairly sure I saw it in my local swagger, I'll take a look

Reply:
I did the process again and uploaded an updated version, it's there now